### ISSUE TITLE
No horizontal scrolling in tileset mode

### ISSUE THREAD

**Issue by janitress** [Fleex255/Abiathar-Support#6]
When in tileset mode and zoomed in a bit one can only scroll vertically but not horizontally.

(All versions)

**Comment by Fleex255** [comment]
This is currently the intended behavior. Does any part of the tileset get cut off when zoomed in? If so, that would indeed be a problem.

**Comment by janitress** [comment]
Yeah, depending on your screen resolution / zoom level, part of the tileset does get cut off.

**Comment by Fleex255** [comment]
Ah. Could you please tell me a zoom level and screen resolution combination that reproduces the problem?

**Comment by janitress** [comment]
For e.g. 683x768 and a zoom level of 300%. 
(683 = 1366/2; tiled screen with two windows opened side by side!)

**Comment by Fleex255** [comment]
Interesting, thanks. I'm still having some trouble reproducing this -- may I see a screenshot?

**Comment by janitress** [comment]
screen resolution: 1366 x 768
windows: two tiled abiathar windows
zoom level left: 100%
zoom level right: 400%
![SAIO](https://s19.postimg.cc/dgv16fw3n/Abiatharx.png)

**Comment by Fleex255** [comment]
Oh, I see. I was fixating on the zoomable simultaneous tileset, since I had been working on that for a while and had made sure it wasn't possible for it to overflow the available space. Sorry about that! Reclassifying this as a feature request to be implemented Eventually(TM) since there's a lot of internal stuff that assumes tileset view states have no X offset.
